<?xml version="1.0"?>
<!-- Example launch file for ECON Camera CU20 -->

<launch>
     <remap from="imu_cam_sync/mag" to="imu/magnetometer"/>

     <arg name="baudrate" default="921600"/>
     <arg name="dev_name" default="/dev/ttyTEL1"/>


     <node pkg="imu_cam_sync" type="imu_cam_sync_node" name="imu_cam_sync_node" output="screen" >
        <param name="dev_name" type="str" value="$(arg dev_name)" />
        <param name="baudrate" type="int" value="$(arg baudrate)" />

     
     
     </node>


   
</launch>